lte/gateway/c/core/oai/tasks/amf/amf_app_handler.cpp

```diff
@@ -1540,12 +1607,398 @@ void amf_app_handle_initial_context_setup_rsp(
          * 2nd time PDU session establish message
          */
         create_session_grpc_req_on_gnb_setup_rsp(
-            &amf_smf_grpc_ies, imsi, smf_context->pdu_session_version);
+            &amf_smf_grpc_ies, imsi, smf_context->pdu_session_version,
+            smf_context);
       }
     }
   }
 
   // update UE state
   ue_context->mm_state = REGISTERED_CONNECTED;
 }
+
+/****************************************************************************
+ **                                                                        **
+ ** Name:    amf_app_pdu_session_modification_request()                    **
+ **                                                                        **
+ ** Description: Send the PDU modification request to gnodeb               **
+ **                                                                        **
+ ** Inputs:  pdu_sess_mod_req:   pdusession modification request           **
+ **      ue_id:      ue identity                                           **
+ **                                                                        **
+ **      Return:    RETURNok, RETURNerror                                  **
+ **                                                                        **
+ ***************************************************************************/
+int amf_app_pdu_session_modification_request(
+    itti_n11_create_pdu_session_response_t* pdu_sess_mod_req,
+    amf_ue_ngap_id_t ue_id) {
```

Comment:
General comment: Add OAILOG_FUNC_IN() and OAILOG_FUNC_RETURN() while entering and exiting the function